lib/spack/docs/environments.rst

```diff
@@ -952,6 +952,17 @@ function, as shown in the example below:
      ^mpi: "{name}-{version}/{^mpi.name}-{^mpi.version}-{compiler.name}-{compiler.version}"
      all: "{name}-{version}/{compiler.name}-{compiler.version}"
 
+Projections also will permit environment variable and spack configuration variable
```

Comment:
```suggestion
Projections also permit environment and spack configuration variable
```

@psakievich this is awesome! Totally optional suggestion to clean up the language a little. 😃